packages/tasks/src/tasks/image-to-3d/data.ts

```diff
@@ -0,0 +1,76 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "A large dataset of over 10 million 3D objects.",
+			id: "allenai/objaverse-xl",
+		},
+		{
+			description: "A dataset of isolated object images for evaluating image-to-3D models.",
+			id: "dylanebert/iso3d",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "image-to-3d-image-input.png",
+				type: "img",
+			},
+		],
+		outputs: [
+			{
+				label: "Result",
+				content: "image-to-3d-3d-output-filename.glb",
```

Comment:
I don't know if we can render glb files, can you put an image of how final 3d image looks like? and maybe put in label that it's a glb file